test/archive_test_results.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+ 
+cd ${WORKSPACE}/test
+
+tar czf test_results_${JOB_BASE_NAME}_${BUILD_NUMBER}.tar test_results/
```

Comment:
Could you create a variable for `test_results` and avoid the hardcoding in lines 5 and 7. Thanks.

Reply:
Good idea, will do :)

Reply:
Fixed!